api/src/pcapi/routes/pro/offers.py

```diff
@@ -863,6 +863,43 @@ def post_price_categories(
     return offers_serialize.GetIndividualOfferWithAddressResponseModel.from_orm(offer)
 
 
+@private_api.route("/offers/<int:offer_id>/price_categories", methods=["PATCH"])
```

Comment:
PUT me semble être plus approprié comme verbe HTTP.
```suggestion
@private_api.route("/offers/<int:offer_id>/price_categories", methods=["PUT"])
```